api/types/entrySchema.go

```diff
@@ -38,7 +39,59 @@ func (s EntrySchema) MarshalJSON() ([]byte, error) {
 	return json.Marshal(s.EntrySchema)
 }
 
-// UnmarshalJSON unmarshals the entry's schema from JSON.
+// UnmarshalJSON unmarshals the entry's schema from JSON. EntrySchemas
```

Comment:
Hmm, I think this could move to the EntrySchema type declaration b/c [fs/schema's response is a different type](https://github.com/puppetlabs/wash/blob/master/api/schema.go#L16).

Reply:
Regardless, I consider the refactor here a bug-fix (even though the previous unmarshalling didn't cause any issues in Wash's normal operation).